toolchains/xslt-M4/document/xml/element-reference-html.xsl

```diff
@@ -124,10 +137,19 @@
          <xsl:variable name="header-class" expand-text="true">{ if (exists(parent::map)) then 'definition' else 'instance' }-header</xsl:variable>
          <div class="{ $header-class }">
             <!-- generates h1-hx headers picked up by Hugo toc -->
-            <xsl:element expand-text="true" name="{ $header-tag }" namespace="http://www.w3.org/1999/xhtml">
-               <xsl:attribute name="id" select="@_tree-xml-id"/>
-               <xsl:attribute name="class">toc{ $level} name</xsl:attribute>
-               <xsl:text>(unwrapped)</xsl:text>
+
+            <!-- ===!!! Anchor hard wrap-around of Headers 1-6 [see logic above with ($level le 6)] !!!=== -->               
+            <xsl:element expand-text="true" name="a" namespace="http://www.w3.org/1999/xhtml">
+               <xsl:attribute name="href">#{@_tree-xml-id}</xsl:attribute>
+               <xsl:attribute name="class">anchor-xslt toc{ $level} name  Elem-Ref-Html--Xsl</xsl:attribute>
```

Comment:
Ditto here.

```suggestion
               <xsl:attribute name="class">reference-element-anchor toc{ $level} name</xsl:attribute>
```

Reply:
Nice review @aj-stein-nist - it's very helpful to me to see a good example.

Reply:
Killed the canary class-names